src/components/Bars/Bars.jsx

```diff
@@ -267,26 +276,30 @@ const Bars = createClass({
 									</ToolTip.Title>
 
 									<ToolTip.Body>
-										<Legend hasBorders={false} isReversed={isStacked}>
-											{_.map(yFields, (field, fieldIndex) => (
-												<Legend.Item
-													key={fieldIndex}
-													hasPoint={true}
-													pointKind={1}
-													color={_.get(
-														colorMap,
-														field,
-														palette[(fieldIndex + colorOffset) % palette.length]
-													)}
-												>
-													{yTooltipFormatter(
-														_.get(legend, field, field),
-														yFormatter(data[seriesIndex][field]),
-														data[seriesIndex][field]
-													)}
-												</Legend.Item>
-											))}
-										</Legend>
+										{tooltipFormatter
```

Comment:
I think this would be a little cleaner if you pulled the default toolTipFormatter up into a `this.defaultToolTipFormatter`, and used that as the default value for `this.props.toolTipFormatter`. What do you think?

Reply:
Makes sense to me!

Reply:
Just gave this refactor a go - `getDefaultProps` doesn't have access to instance variables so can't reference `this.defaultTooltipFormatter` there. I could either define the default formatter within the `getDefaultProps` scope and add a `props` argument (since we don't have access to `this.props` in that scope) or leave it as is... maybe with a small refactor like: ```{tooltipFormatter ? tooltipFormatter(data[seriesIndex]) : defaultTooltipFormatter(data[seriesIndex])}```

Will defer to your preference on this! Or maybe you see a cleaner way to deal with this that I don't.